projects/angular/src/data/datagrid/render/main-renderer.ts

```diff
@@ -136,6 +151,9 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, Af
 
   ngOnDestroy() {
     this.subscriptions.forEach(sub => sub.unsubscribe());
+    if (this.intersectionObserver) {
+      this.intersectionObserver.disconnect();
```

Comment:
```suggestion
      this.intersectionObserver?.disconnect();
```

This can be simplified. The `if` isn't needed.